src/pages/App.tsx

```diff
@@ -112,7 +112,7 @@ function App(): JSX.Element {
                         ref={bodyRef}
                         className="flex flex-col items-center justify-between flex-grow h-full overflow-y-auto overflow-x-hidden z-0 pt-4 sm:pt-8 px-4 md:pt-10 pb-4"
                         style={{
-                            background: theme.color.mainBg
+                            background: 'white'
```

Comment:
why not using theme?

Reply:
we had a type issue with the theme object so for now decided to ignore and fix later

Reply:
ok but how would you know you need to get back to this specific place?
you must mark it with a TODO or something

Reply:
OK, will update this in the next feature branch